### PR TITLE
Species model

### PR DESCRIPTION
**Pull request by JorgeATR**
### What does this PR do?

* Created the species model
* Created Factory, RSpec tests and model validations for the model

### CODE REVIEW

File: app/models/species.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Species < ApplicationRecord
```
Comment (kurenn) [inline app/models/species.rb]: Models are NEVER on plural, please update it

File: spec/factories/species.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FactoryGirl.define do
+  factory :species do
+    name {FFaker::Lorem.paragraph}
+    description {FFaker::Lorem.paragraph}
+    source_page {(rand * 100).to_i}
+    source_book {"Force and Destiny Core Rulebook"}
+    brawn {1}
```
Comment (kurenn) [inline spec/factories/species.rb]: Why using the `{}`?

File: spec/factories/species.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+FactoryGirl.define do
+  factory :species do
+    name {FFaker::Lorem.paragraph}
```
Comment (kurenn) [inline spec/factories/species.rb]: Put a space between the instruction inside and the `{}`, something like:

```ruby
name { FFaker::Lorem.paragraph }
```

**Review by kurenn — CHANGES_REQUESTED** [review]

File: spec/factories/species.rb (hunk 2 above)
Comment (JorgeATR) [inline spec/factories/species.rb]: `source_page {(rand * 100).to_i}`
After removing the unnecessary {}'s this line throws "Undefined * method", can I leave that line with curly braces or is there another way to do it?

**Pull request merged**